ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift

```diff
@@ -6,6 +6,43 @@
 import FluentUI
 import UIKit
 
+public final class DemoBarButtonItems: NSObject {
+    static func staticImageNamed(_ name: String) -> UIImage? {
```

Comment:
we don't need this function. follow other examples of getting images

Reply:
The function 'staticImageNamed' and all its containing class has been removed

Reply:
Removed function 'DemoBarButtonItems' and all its containing class